students/TinaB/lesson05/mailroom_part3.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+"""
+$ chmod +x mailroom_part3.py
+"""
+import datetime
+import os
+
+DONORS_DICT = {"William Gates": [326892.24, 122, 22, 12],
+               "Mark Zuckerberg": [30, 60, 65982.55],
+               "Jeff Bezos": [52636.27],
+               "Paul Allen": [877.33, 22],
+               "Steven Hawking": [326892.24, 123, 123.33, 123, 123],
+               "Justin Timberlake": [999658.25, 1233, 123]}
+
+# Menus
+
+
+def menu_selection(prompt, dispatch_dict):
+    """menu function"""
+    try:
+        while True:  # loop forever until they quit
+            response = input(prompt)
+            if dispatch_dict[response]() == "exit menu":
+                break
+    except KeyError:
+        print("Invalid input, quitting program")
+
+
+def single_print_sub_menu():
+    '''Sub Menu for Print single Thank you menu'''
+    menu_selection(SINGLE_PRINT_SUB_PROMPT, SINGLE_PRINT_SUB_DISPACTH)
+
+
+def quit_menu():
+    '''Quit menu function and method'''
+    print("Quitting this menu now")
+    return "exit menu"
+
+
+# functions for prompts ----
+
+def get_name_input():
+    ''' Function to select user input to return to print function'''
+    name_input = input(
+        "Please enter the name of the donor:  ")
+
+    for donor, donations in DONORS_DICT.items():
```

Comment:
you never use `donations` var, if you only need keys: `for donor in donors_dict:`